### ISSUE TITLE
Hello from Apple Coding, Being a co-producer, or producing an episode

### ISSUE THREAD

**Issue by applecoding** [SwiftCommunityPodcast/podcast#21]
Hello,

My name is Julio César Fernández, host and producer of Apple Coding podcast (https://cuonda.com/apple-coding) that is related to technology through the development, and focused on Apple technologies and Swift. I'm the author of "Aprendiendo Swift", a book to learn Swift in spanish with more than 2.500 copies sold. Also I'm a Swift trainer with more than 3 years of experience and usual guest at Applelianos podcast where Gui Rambo was invited as a guest a couple of times.

Will be a pleasure to help this project about produce or help about how to begin in Swift, why Swift above other languages and other things about learning language and bringing new developers into the language.

**Comment by garricn** [comment]
Glad to hear you you're interested in being involved! We're currently testing out a process for empowering the community to create episodes for the podcast. Since you have experience producing a podcast, you're feedback on the following PR's would be greatly appreciated: #18 #19 #22.

**Comment by applecoding** [comment]
Great. Thanks Garric. It will be great to collaborate to define the production helping with my experience. Best regards.

**Comment by garricn** [comment]
> Great. Thanks Garric. It will be great to collaborate to define the production helping with my experience. Best regards.

We’re you able to take a look at #18 #19 and #22 ? Also, feel free to create your own Episode Proposal PR by following the guidelines set out in those PRs.

**Comment by applecoding** [comment]
Sure. I am working on a new Episode Proposal PR and i’ll the topics

**Comment by lattner** [comment]
I would love for you to get involved @applecoding!  Once we get more basic issues resolved and get established, I'd also love to hear your thoughts about how we can better reach more non-english speaking folks.

**Comment by lattner** [comment]
Thank you for the kind words, I hope you can get involved in this project, please let us know!

**Comment by applecoding** [comment]
Of course. You can count with me. I’m preparing various proposals for episodes and I will be active on the threads to help the project as much as I can. Thanks
